services/movie_session.py

```diff
@@ -42,3 +42,12 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
The `get` method can raise a `DoesNotExist` exception if no `MovieSession` with the given `id` is found. Consider using `get_object_or_404` or adding exception handling to manage this scenario.